Added option to pack build output into tarball

Miniwa opened the pull request:
Like the title says, I added a 'pack' task to gulp that creates a tarball of the output.
Makes it easier to access and deploy the output using tools for automated release.

aldergames commented:
Is there a similar one for .zip? That's usually the format I pack things in, as it's one that more people will know what the hell it is.

Miniwa commented:
Yeah maybe you're right.
I added zip output as well and split the different formats into individual tasks.
The 'pack' master task now generates both a zip and a tarball though, I dont know if that's confusing since users might think they need both or some shit.
